code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -78,6 +79,18 @@
 	ignore_flags = 1 // So they can heal their comrades.
 	list_reagents = list(/datum/reagent/medicine/epinephrine = 30, /datum/reagent/medicine/omnizine = 30, /datum/reagent/medicine/leporazine = 15, /datum/reagent/medicine/stabilizing_nanites = 15)
 
+/obj/item/reagent_containers/hypospray/combat/weaker
+	name = "combat stimulant injector"
+	desc = "A modified air-needle autoinjector, used by syndicate operatives to quickly patch their own wounds in combat."
+	amount_per_transfer_from_this = 15
+	item_state = "combat_hypo"
+	icon_state = "combat_hypo"
+	volume = 90
+	possible_transfer_amounts = list(10,15,30,45)
+	self_only = 1
```

Comment:
It sill inherit the armour peircing ability from the combat hypospray
```suggestion
	self_only = 1
	ignore_flags = 0
```

Reply:
oop, missed that